### DISCUSSION TITLE
Is there an abstraction to have a list of records always up to date from similar to the Firebase's onSnapshot?

### DISCUSSION THREAD

**Discussion by joaomelo** [nhost/nhost#1113] [Q&A] [answered]
Hi there,

Is there an abstraction to keep a list of records from a table in sync, similar to the Firebase's [onSnapshot ](https://firebase.google.com/docs/firestore/query-data/listen#listen_to_multiple_documents_in_a_collection)?

I could not find this in the NHost docs.

Thanks in advance.

**Comment by elitan** [comment, marked as answer]
Yes, it's possible, and it's called **streaming subscriptions** in Hasura/Nhost.

How it works: https://hasura.io/docs/latest/subscriptions/postgres/streaming/index/

Example: https://hasura.io/docs/latest/subscriptions/postgres/streaming/use-cases/

**Reply by joaomelo** [reply]
thanks @elitan

**Reply by elitan** [reply]
My pleasure.
